### PR TITLE
Slack Plugin demo

### PR DESCRIPTION
**Pull request by victorsalaun**
Add demo for Slack Plugin

**Do not merge** till the code of this PR https://github.com/jenkinsci/slack-plugin/pull/404 is added to a new release of the plugin

### CODE REVIEW

File: demos/slack/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Configure Slack
+
+Basic configuration of the [Slack Plugin](https://wiki.jenkins.io/display/JENKINS/Slack+Plugin)
+
+## Sample configuration
+
+```yaml
+unclassified:
+  slackNotifier:
+    baseUrl: https://workspace.slack.com/services/hooks/jenkins-ci/
```
Comment (timja) [inline demos/slack/README.md]: I wouldn’t add the base url in the default sample, that’s only for slack compatible apps, teamDomain is the important one for most people

Comment (timja) [reply demos/slack/README.md]: there's a PR on the slack plugin to make the wording better on the help message for it,
slack compatible means apps like mattermost,
people actually using slack should use teamDomain

Here's the example I added to the readme of the slack plugin:
https://github.com/jenkinsci/slack-plugin#configuration-as-code

**Comment by victorsalaun** [comment]
Thanks for the reference @timja I copied your example here

File: demos/slack/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+              scope: GLOBAL
+              id: slack-token
+              secret: '${SLACK_TOKEN}'
+              description: Slack token
+
+
+unclassified:
+  slackNotifier:
+    teamDomain: <your-domain>
+    tokenCredentialId: <secret-text-token>
```
[the remark this answers is not in the document]
Comment (timja) [reply demos/slack/README.md]: @victorsalaun are you able to up date this? the plugin is now released

**Comment by timja** [comment]
@ndeloof you can remove the on-hold label now

File: demos/slack/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (victorsalaun) [reply demos/slack/README.md]: thanks @timja I will do it this today!

[the remark this answers is not in the document]
Comment (victorsalaun) [reply demos/slack/README.md]: @timja copy paste your example done :)

**Comment by timja** [comment]
@ewelinawilkosz could you merge please 👍

**Comment by ewelinawilkosz** [comment]
yes, sorry @timja - missed the notification, will take care of that today

**Pull request merged**